frontend/src/global_styles/content/editor/_ckeditor.sass

```diff
@@ -41,6 +41,10 @@ ckeditor-augmented-textarea .op-ckeditor--wrapper
     th[style^="width:"]
       word-break: break-all
 
+  .table table th
+    border-color: var(--borderColor-default) !important
+    background-color: var(--bgColor-muted) !important
```

Comment:
Please get rid of the `!important`. There are ways to be more specifc and thus enforce our styles without that. Further, this style is duplicating the user-content table header styles. So please extract that into a mixin. Otherwise, one will somewhen change and the other will be forgotten